docs/amber.md

```diff
@@ -0,0 +1,138 @@
+# Amber
+
+Amber is a multi-API shader test framework.Graphics bugs can be captured and
+communicated through a scripting interface. This removes the need to program
+to the low level interface when reproducing bugs.
+
+Amber is broken into multiple parts, the applications, the parsing components
```

Comment:
recommend change the first comma to a colon.

Reply:
Should we change parts --> layers ?   they truly are layers.

Reply:
done. layers is a better name, thanks.